js/script.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+$(function() {
+
+    function checkUser(e, elements, users) {
+        let active = elements.activeName;
+        if(e.keyCode === 40){
+            if(elements.isOpen && active >= 0 && active < 9){
```

Comment:
@lianatumikyan maybe here you can write inline 'if' like this
elements.activeName = elements.isOpen && active >= 0 && active < 9 ? active + 1 : 0;